src/views/Hearing/Sidebar.js

```diff
@@ -102,17 +104,29 @@ class Sidebar extends React.Component {
               <div className="sidebar-section contents">
                 <h4><FormattedMessage id="table-of-content"/></h4>
                 <ListGroup>
-                  <ListGroupItem href="#hearing">
+                  {!isQuestionView && <ListGroupItem className={currentlyViewed === '#hearing' && 'active'} href="#hearing">
```

Comment:
Hm, Ville had already added the subsections in the contents list? What happened to that?